src/Main.java

```diff
@@ -1,15 +1,68 @@
-//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
-// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-        //TIP Press <shortcut actionId="ShowIntentionActions"/> with your caret at the highlighted text
-        // to see how IntelliJ IDEA suggests fixing it.
-        System.out.printf("Hello and welcome!");
-
-        for (int i = 1; i <= 5; i++) {
-            //TIP Press <shortcut actionId="Debug"/> to start debugging your code. We have set one <icon src="AllIcons.Debugger.Db_set_breakpoint"/> breakpoint
-            // for you, but you can always add more by pressing <shortcut actionId="ToggleLineBreakpoint"/>.
-            System.out.println("i = " + i);
+
+        //task1
+        System.out.println("");
+        char clientOS = 0;
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+
+        //task2
+        System.out.println("");
+        int clientDeviceYear = 2015;
+        if (clientDeviceYear >= 2015) {
+            if (clientOS == 0) {
+                System.out.println("Установите версию приложения для iOS по ссылке");
+            } else {
+                System.out.println("Установите версию приложения для Android по ссылке");
+            }
+        } else if (clientOS == 0) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else {
+            System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        }
+
+        //task3
+        System.out.println("");
+        int year = 2024;
+        int everyFourth = year % 4;
+        int everyHundredth = year % 100;
+        if (year > 1584) {
+            if ((everyFourth == 0) && (everyHundredth != 0)) {
+                System.out.println("Год високосный");
+            } else {
+                System.out.println("Это невисокосный год");
+            }
+        }
+
+        //task4
+        System.out.println("");
```

Comment:
```suggestion
        System.out.println();
```
зачем пустая строка передаётся?